CONTRIBUTING.md

```diff
@@ -374,3 +374,20 @@ We use [RenovateBot](https://renovatebot.com/) to automatically upgrade our depe
 We will try to review and merge pull requests quickly. After merging we
 will try releasing a new version. If you want to know our build process or
 build your own Cypress binary, read [DEPLOY.md](DEPLOY.md)
+
+## Known problems
+
+### ENFILE
```

Comment:
I believe you mean `EMFILE`, not `ENFILE`.

This should never happen as long as we *never* use synchronous FS methods - which I'm pretty sure we don't. This is what `graceful-fs` fixes.

If you are seeing this it means we messed something up and need to understand where it's happening and fix it.

Reply:
this was during the build

Reply:
turns out to be sync file system bug - #4448